css/styles.css

```diff
@@ -14,7 +14,7 @@ body.landing-page {
    color: #000;
    font-size: 12px;
    line-height: 1.4;
-   font-family: Helvetica, Arial, sans-serif;
+   font-family: 'Open Sans', sans-serif;
    background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
```

Comment:
the pictures are not rendering and I think it might be written as background: url( images/header_bg.jpg)